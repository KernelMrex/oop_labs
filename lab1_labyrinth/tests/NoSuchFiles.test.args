bad nosuchfile1 nosuchfile2
