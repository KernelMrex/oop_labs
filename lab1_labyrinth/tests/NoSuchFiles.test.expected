Error while reading labyrinth: couldn't open file
